### ISSUE TITLE
Crash to Launcher Before Startup 

### ISSUE THREAD

**Issue by TalhaErenY** [FalsePattern/FalseTweaks#182]
Modlist:
+unimixins-all-1.7.10-0.1.17
falsepatternlib-mc1.7.10-1.2.5
falsetweaks-mc1.7.10-2.8.1

Forge Log: https://pastebin.com/XVuu2Zvt

Config: https://pastebin.com/bMeGBDjG

Issue description: attempting to start up the game with any config module set to true will result in a crash to launcher with exit code 1 (vanilla launcher is being used).

No JVM arguments, default java install of the launcher. setup works fine for mods like optifine and archaic fix on their own.

**Comment by FalsePattern** [comment]
FalseTweaks 2.8 is no longer supported, try again with the 3.0 beta

**Comment by TalhaErenY** [comment]
Apologies, I neglected to mention I had already tried the 3.0.0-pre1 build first, which also resulted in the same error code and crash.

**Comment by FalsePattern** [comment]
Please update to the latest version of java 8, you're using an extremely old version of java that doesn't work correctly with mixins.

And also make sure to use a modern modded minecraft launcher, such as PrismLauncher, playing with mods through the vanilla or curseforge launchers is extremely unstable with newer mods, as they both use a very old internal version of java.

**Comment by TalhaErenY** [comment]
I can confirm that on a fresh install of prism, with j8-412 manually downloaded and minecraft pointed at, it does work. It still does not work with the vanilla launcher (despite the newer download of j8), or another launcher I tried. I suppose I'll migrate to prism for the time being. I failed to mention previously that I had already tried the vanilla launcher with j8-412, though when it didn't work I used a fresh instance to reduce variables from using an already modded instance (then promptly forgot to manually point it to the correct java install.)
